libs/media-player/src/lib/shared/as-two-digit-string.ts

```diff
@@ -0,0 +1,19 @@
+import { isInteger, isNonNegativeFiniteNumber } from '@coscrad/validation-constraints';
+
+/**
+ *
+ * @param inputNumber non-negative finite number less than 100
+ * @returns string
+ */
+
+export const asTwoDigitString = (inputNumber: number) => {
+    if (!isNonNegativeFiniteNumber(inputNumber) || inputNumber > 99) {
+        throw new Error('inputNumber must be a non-negative finite number that is less than 100');
+    }
+
+    const inputAsString = inputNumber.toString();
+
+    if (!isInteger(inputNumber)) return inputAsString;
```

Comment:
Don't do this. If the input is 99.454 the output will be "99.454", which is not a two-digit-string.